week3project/main.js

```diff
@@ -14,16 +14,13 @@ backarrow.addEventListener("click", function(){
 
 inputfield.addEventListener("keyup", function(e){
 if(e.key === "Enter" && inputfield.value != ""){
-    requestApi(inputfield.value);       
+    requestApi(inputfield.value); 
+    setTimeout(() => {
+        var paragraph = document.createElement("p");
+        paragraph.innerText = inputfield.value;
+         searchhistory.appendChild(paragraph);
+  }, 3000);      
```

Comment:
The problem earlier still exists. Adding to the history will still happen regardless of if the API request fails or succeeds. We only want to add it if the search succeeds